web/vocab/TERMS.md

```diff
@@ -0,0 +1,109 @@
+
+## ExperienceEvent
+- A particular Event recorded by a Person to describe an experience they had and how it connects to religion in the area.  Aggregates the following concepts:
+     - Sense
+     - Practice
+     - Object
+     - Person
+     - Researcher
+     - Place
+
+## Event
+- A place and time at which some action was observed or is known to have happened.  See http://schema.org/Event
+
+## Place
+- A location, made knowable by some label (metaphysical, imaginary or not on planet Earth, ex. "Legion of Doom") and/or earthbound coordinates.  See http://schema.org/Place.
+
+## Person
+- An imaginary or real human being.  See http://schema.org/Person
+
+## Researcher
+- All the qualities of a Person except information is limited to name only and the context is confined to partners in the Lived Religion project.
+
+## Sense
+- One of the five common human senses: Sight, Sound, Smell, Touch, Taste.  See http://id.worldcat.org/fast/1112562
+
+## Practice
+- An action being performed by individual(s) based on religious guidance.  See http://id.worldcat.org/fast/1149637
+
+## Object
+- Observable physical matter. See http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#PhysicalResource
+
+## RelatedSenses
+- Aggregates the various Senses under an ExperienceEvent.
+
+## RelatedPractices
+- Aggregates the various Practices under an ExperienceEvent.
+
+## RelatedObjects
+- Aggregates the various Objects under an ExperienceEvent.  See http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#hasArtefactRelatedPhysicalResource
+
+## Sight
```

Comment:
These senses... I feel like something is off, but I'm not sure how to pin it down...

"Noting the person" is rules about expected properties, not a definition, so that isn't required. I think, though, the way this will be used and is framed, these are not a person's actual sense of x, but the sensation itself. I mean, the Sight of Person is NOT what is part of the Experience, but that the Sight (seeing of the array of candles) witnessed by Person is related to the Experience.

Reply:
Yeah.  Again, how we ultimately aggregate these concepts under an Experience as a subclass of Event is up in the air a bit.  These were mostly encoded as a way to say `There is a specific aggregator under Experience like 'Experience.sensory[]' that encapsulates the Senses encountered at the Experience.`

Reply:
Yes. You're language diagrams to what I stated. The `sensory` property is a container for the related sensations, not "Bryan's sense of vision" and "Patrick's sense of smell" as `Sight` is currently defined.